src/api/app/controllers/webui/users/bs_requests_controller.rb

```diff
@@ -15,16 +15,24 @@ class BsRequestsController < WebuiController
       }.freeze
 
       def index
-        if Flipper.enabled?(:request_index, User.session)
-          filter_requests
+        respond_to do |format|
+          format.html do
+            filter_requests
+
+            # TODO: Temporarily disable list of creators due to performance issues
+            # @bs_requests_creators = @bs_requests.distinct.pluck(:creator)
+            @bs_requests = @bs_requests.order('number DESC').page(params[:page])
+            @bs_requests = @bs_requests.includes(:bs_request_actions, :comments, :reviews)
+            @bs_requests = @bs_requests.includes(:labels) if Flipper.enabled?(:labels, User.session)
+          end
```

Comment:
This belongs to `request_index` only.

Reply:
It does, however if we want to avoid errors when a user refreshes after a logout for instance, we can't just selectively render different formats, we would be much better off handling that as a redirect in a before action instead

Reply:
I implemented a redirect

Reply:
You have redirect to `my_tasks_path` but I believe this action is used for both `my tasks` and `/requests`

Reply:
It's used for my_tasks' datatable, so I can still redirect to it

Reply:
If someone is logged out should they see this page at all?

Reply:
Sure, logged out example is bad, but if you switch beta state and then refresh, or if somebody sends you a link to this page, you are going to have the same issue